### PR TITLE
[WORKAROUND]: run physxlegacy setup without user input

### PR DESCRIPTION
**Pull request by imLinguin**
As we noticed in https://github.com/Heroic-Games-Launcher/known-fixes/pull/7  
the PHYSXLEGACY setup doesn't have a silent switch, or at least it doesn't respect it.

I came up with this simple solution, it should make the experience more hands free.

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Review by arielj — APPROVED** [review]
Works as expected. Tested with Empire Earth 3 with and without this change.

With this change physx legacy is installed silently.

**Review by flavioislima — APPROVED** [review]
LGTM!

Not sure if this could be called a hack, you are just adapting the command to what the dependency needs.

If was the case that we would need to handle several more dependencies with different commands then we could use a mapping object for it.

For me this is just a normal fix and I would remove the Hack commend before merging. 
But I will leave it up to you 👍🏽

**Comment by imLinguin** [comment]
Maybe a workaround is better suited for this

**Comment by imLinguin** [comment]
Maybe... but here we are branching out from the standard way even more 😄

**Pull request merged**
